### PR TITLE
feat: Address smaller UI issues in the new navigation (WPB-9694)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9694" title="WPB-9694" target="_blank">TaskWPB-9694</a>  Fix smaller UI issues in the new navigation 
  </td></table>
  <br />
 

#### Overview
This pull request addresses a collection of minor UI issues in the new navigation that need to be resolved before the public release. The changes aim to enhance the overall user experience and visual consistency of the navigation interface.

#### Changes Implemented
1. **Icon Button for "+Group"**
   - Changed the “+Group” button to an icon button.
   - Removed “Group” from the label.

2. **Consistent Font Style**
   - Applied the same font style for:
     - Display name
     - Title in the conversation list
     - Title in the conversation title

3. **Avatar Padding**
   - Added 16px padding to the top of the user avatar in both collapsed and expanded states.

4. **Sidebar Collapse/Expand Button**
   - Made the sidebar collapse/expand button visible only on hover in both collapsed and expanded states.

5. **Icon for “Connect” Item**
   - Changed the icon of the “connect” item to “Add participants”.

6. **Updated Sidebar Icons**
   - Updated the icon for the folder in the sidebar.
   - Added an arrow for collapsing and expanding the folder section (design specs referenced).

7. **Support Link Icon**
   - Added an icon to the support link in the sidebar to indicate it opens an external link.

#### Screenshots
1. **Before and After for "+Group" Button**
image

image

2. **Font Style Consistency in Conversation List and Title**

image

3. **User Avatar Padding**

image

4. **Sidebar Collapse/Expand Button Hover State**

image
image

5. **Updated "Connect" Item Icon**

image

6. **Folder Icon and Arrow in Sidebar**

image

7. **Support Link Icon**

image

### CODE REVIEW

File: src/script/page/LeftSidebar/UserDetails/UserDetails.tsx [hunk 1]
```diff
@@ -115,6 +115,7 @@ const UserDetailsComponent = ({user, isTeam = false, groupId, isSideBarOpen = fa
               tabIndex={TabIndex.FOCUSABLE}
               title={userName}
             >
+              77777777
```
Comment (PatrykBuniX) [inline src/script/page/LeftSidebar/UserDetails/UserDetails.tsx]: A leftover?

File: src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx [hunk 2]
```diff
@@ -113,8 +113,6 @@ export const ConversationHeader = ({
             css={button}
           >
             <Icon.PlusIcon />
-
-            {t('conversationGroupCreate')}
```
Comment (PatrykBuniX) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx:117]: Maybe we can include it as button's `title` prop?

File: src/script/page/LeftSidebar/UserDetails/UserDetails.tsx (hunk 1 above)
Comment (V-Gira) [reply src/script/page/LeftSidebar/UserDetails/UserDetails.tsx]: @thisisamir98 's lucky number :D

File: src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx (hunk 2 above)
Comment (V-Gira) [reply src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx:117]: I went with a more relevant string (the previous one is just "Group")
image

**Review by PatrykBuniX — APPROVED** [review]

**Pull request merged**